Dockerfile-python

```diff
@@ -1,5 +1,24 @@
-FROM python:3
+FROM python:3.5
+RUN pip install togeojsontiles
 RUN pip install postgres
 RUN pip install psycopg2
```

Comment:
Let's move all the dependencies in a requirements.txt file and do `pip install -r requirements.txt` here.